src/lib/y2users/user.rb

```diff
@@ -113,6 +113,13 @@ def ==(other)
       ATTRS.all? { |a| public_send(a) == other.public_send(a) }
     end
 
+    # Whether it is the root user
+    #
+    # @return [Boolean]
+    def root?
+      name == "root"
```

Comment:
that reminds me another trick to prevent ssh attacks...rename root to toor, but I expect it is really corner case and as we have this only for installation, it should not be issue.